browser/ui/webui/brave_adblock_internals_ui.cc

```diff
@@ -0,0 +1,143 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "brave/browser/ui/webui/brave_adblock_internals_ui.h"
+
+#include <memory>
+#include <utility>
+#include <vector>
+
+#include "base/bind.h"
+#include "base/process/process.h"
+#include "base/strings/string_number_conversions.h"
+#include "base/values.h"
+#include "brave/browser/brave_browser_process.h"
+#include "brave/browser/ui/webui/brave_webui_source.h"
+#include "brave/components/brave_adblock/adblock_internals/resources/grit/brave_adblock_internals_generated_map.h"
+#include "brave/components/brave_shields/browser/ad_block_service.h"
+#include "components/grit/brave_components_resources.h"
+#include "content/public/browser/web_ui.h"
+#include "content/public/browser/web_ui_controller.h"
+#include "content/public/browser/web_ui_data_source.h"
+#include "content/public/browser/web_ui_message_handler.h"
+#include "services/resource_coordinator/public/cpp/memory_instrumentation/memory_instrumentation.h"
```

Comment:
pls ensure dependencies you use here are reflected in gn files. `browser/ui` has `check_includes = false`, which makes it pretty scary to add stuff like this without manual deps revision.

Reply:
Thanks, done.